docs/data/tree-view/overview/overview.md

```diff
@@ -75,7 +75,29 @@ Or increasing the width of the item state indicator to be full-width:
 
 ## Customization
 
-### Custom icons, border and animation
+### Custom icons
+
+You can customize the collapse, expand and end icons of the TreeView.
+
+You can use icon liraries:
+
+{{"demo": "CustomMaterialIcons.js", "defaultCodeOpen": true}}
+
+Or you can create your own components:
+
+{{"demo": "CustomIcons.js", "defaultCodeOpen": false}}
+
+### Adding borders
```

Comment:
```suggestion
### Connection border
```

Not sure if this is the best title, but felt like we could be clearer with what "borders" means in the context of the TreeView! Open to other suggestions 😬

Reply:
Connection border**s** maybe? But other than that, I don't have an idea 🤔